LDMP/models/datasets.py

```diff
@@ -91,13 +91,21 @@ def child(self, row) -> None:
 class Dataset(DatasetBase):
 
     dumped = pyqtSignal(str)
+    downloaded = pyqtSignal(str)
+    class Origin(Enum):
```

Comment:
I would prefer to define the Origin enum outside the `Dataset` class

Reply:
rationale?

Reply:
We should have one place where all the enums are defined, eg in the dataset model class.

Reply:
I would agree... but the other enum (DatasetStatus) will be removed because unused (as described in the TODO) ... just leaved in case we change idea.
I put the enum in the class just to leave clear the contect of the origin (e.g. is Dataset), otherwise I should name it DatasetOrigin, and IMHO it's more contextual aware having it accessible as class variable of the Dataset class. 
BTW I'm ok also to move outside if this is important for you.
Let mw knoe your decision.... it's up to you.